site/apache.md

```diff
@@ -0,0 +1,460 @@
+
+# Load Balancing with Apache Web Server
+
+This page describes how to setup and start a Apache Web Server for load balancing inside a Kubernets cluster. The configuration and startup can either be automatic when you create a domain using the WebLogic Operator's `create-weblogic-domain.sh` script, or manually if you have an existing WebLogic domain configuration.
+
+## Build Docker Image for Apache Web Server
+
+You need to prepare the Docker image for Apache Web Server that enbeds Oracle WebLogic Server Proxy Plugin.
```

Comment:
You need to build the

Reply:
I used "prepare" since the steps include download, build and tag, not just build.

Reply:
What about "You need to have the..."?

Reply:
Just noticed a typo here, enbeds, embeds.

Reply:
prepare is fine